version.tf

```diff
@@ -1,12 +1,14 @@
 terraform {
-  required_version = ">= 1.3.0"
-  # If your module requires any terraform providers, uncomment the "required_providers" section below and add all required providers.
-  # Each required provider's version should be a flexible range to future proof the module's usage with upcoming minor and patch versions.
+  required_version = ">= 1.9.0"
 
-  #  required_providers {
-  #    ibm = {
-  #      source  = "IBM-Cloud/ibm"
-  #      version = ">= 1.64.0, < 2.0.0"
-  #    }
-  #  }
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">=1.70.1"
```

Comment:
Version constraint upper bound should be provided i.e. `version = ">= 1.70.1, < 2.0.0"`

Reply:
added